### PR TITLE
Fix UZ locale translation and add tests

### PR DESCRIPTION
**Pull request by gladuz**

### CODE REVIEW

**Comment by lucasmichot** [comment]
Can you also add tests for uz locale @gladuz ?

**Comment by gladuz** [comment]
@lucasmichot Yes I can!

File: tests/Localization/UzTest.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+use Tests\AbstractTestCase;
+
+class UzTest extends AbstractTestCase
+{
+    public function testDiffForHumansUsingShortUnitsUzbek()
+    {
+        Carbon::setLocale('uz');
+
+        $scope = $this;
+        $this->wrapWithTestNow(function () use ($scope) {
```
Comment (lucasmichot) [inline tests/Localization/UzTest.php]: This should be `wrapWithNonDstDate`

**Comment by lucasmichot** [comment]
@briannesbitt for some reasons the Travis statuses don't appear anymore on the PR, even if they are run... : https://travis-ci.org/briannesbitt/Carbon/pull_requests

Maybe something to fix there ?

**Comment by briannesbitt** [comment]
Yes, needed to confirm new permissions.

**Pull request merged**